src/components/Calculator.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable indent */
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] Since you are using a class-based app no need to disable those eslints.

Reply:
Thank you @dasileker